docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -172,46 +188,93 @@ Use the latest version of [Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_
+   1. Add a dashboard variable on top of your dashboard to be able to switch between classic histograms and native histograms. Currently there's no direct support for selectively enabling and disabling queries in Grafana ([issue 79848](https://github.com/grafana/grafana/issues/79848)), however there's a relatively easy workaround. Let the variable take the values -1 or 1 and call it for example `latency_metrics`. Add the following two queries to the panel:
 
-{{< code >}}
+      ```
+      <classic_query> < ($latency_metrics * +Inf)
+      ```
 
-```go
-histogram := prometheus.NewHistogram(
-   prometheus.HistogramOpts{
-      Name: "request_latency_seconds",
-      Help: "Histogram of request latency in seconds",
-      Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100},
-      NativeHistogramBucketFactor: 1.1,
-      NativeHistogramMaxBucketNumber: 100,
-      NativeHistogramMinResetDuration: 1*time.Hour,
-})
-```
+      and separately
 
-```java
-static final Histogram requestLatency = Histogram.build()
-     .name("requests_latency_seconds")
-     .help("Histogram of request latency in seconds")
-     .classicUpperBounds(0.005, 0.01, 0.025, 0,05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 100, Double.NaN);
-     .nativeInitialSchema(3)
-     .nativeMaxNumberOfBuckets(100)
-     .nativeResetDuration(1, TimeUnit.HOURS)
-     .register();
-```
+      ```
+      <native_query> < ($latency_metrics * -Inf)
+      ```
 
-{{< /code >}}
+      Where `classic_query` is the original query and `native_query` is the same but using native histogram query syntax. This technique is employed for Mimir's own dashboards, see for example the [Overview dashboard](https://github.com/grafana/mimir/blob/main/operations/mimir-mixin-compiled/dashboards/mimir-overview.json).
+
+      This solution allows users to switch between the classic histogram and the native histogram without going to a different dashboard.
+
+   1. Replace the existing classic queries with a modified query:
+
+      ```
+      <classic_query>
+      ```
+
+      replaced with
+
+      ```
+      <native_query> or <classic_query>
+      ```
+
+      Where `classic_query` is the original query and `native_query` is the same but using native histogram query syntax.
+
+      {{< admonition type="warning" >}}
+      Using the PromQL operator `or` can lead to unexpected results. For example if the query uses a range of 7 days, such as `sum(rate(http_request_duration_seconds[7d]))` then this query will return a value as soon as there are two native histograms samples present before the end time of the query, thus the 7 day rate will be calculated from a couple of minutes worth of data and not 7 days. This means that the result will be very inaccurate around the time when native histograms were started to be scraped and this inaccuracy will always be there when looking at the graph for that time.
```

Comment:
```suggestion
      Using the PromQL operator `or` can lead to unexpected results. For example, if a query uses a range of seven days, such as `sum(rate(http_request_duration_seconds[7d]))`, then this query returns a value as soon as there are two native histograms samples present before the end time specified in the query. In this case, the seven day rate is calculated from a couple of minutes, rather than seven days, worth of data. This results in an inaccuracy in the graph around the time you started scraping native histograms.
```